Pods/Target Support Files/Pods-IGListKit-tvOSTests/Pods-IGListKit-tvOSTests-frameworks.sh

```diff
@@ -94,7 +94,7 @@ install_dsym() {
     binary="${DERIVED_FILES_DIR}/${basename}.framework.dSYM/Contents/Resources/DWARF/${basename}"
 
     # Strip invalid architectures so "fat" simulator / device frameworks work on device
-    if [[ "$(file "$binary")" == *"Mach-O dSYM companion"* ]]; then
+    if [[ "$(file "$binary")" == *"Mach-O "*"dSYM companion"* ]]; then
```

Comment:
What's the reason for all these script changes?

Reply:
Since these are generated I guess that's because of the CocoaPods update.

Reply:
Well, I'm not guessing, I know for sure.

Reply:
gotcha, so it's Pod generated script, looks good to me then!